config/i18n-tasks.yml

```diff
@@ -6,3 +6,8 @@ data:
   yaml:
     write:
       line_width: -1
+ignore_unused:
```

Comment:
Why are we ignoring these rather than removing them? Could you explain in the commit? (Especially as they're then not removed in the subsequent "Remove unused keys" commit).

My guess is that they actually _are_ used, but the static analysis doesn't spot that.

Reply:
We're ignoring them as they are actually used, but i18n-tasks default unused locale scanners can't detect them. Adding a list prevents i18n-tasks from removing those keys when we run destructive commands like: `i18n-task remove-unused`.